unstoppable-ios-app/domains-manager-ios/Entities/Wallets/WalletEntity.swift

```diff
@@ -47,11 +50,58 @@ extension WalletEntity {
             let domains = createMockDomains()
             let numOfNFTs = Int(arc4random_uniform(10) + 1)
             let nfts = (0...numOfNFTs).map { _ in  NFTDisplayInfo.mock() }
+            
+            
+            let balance: [WalletTokenPortfolio] = [.init(address: $0.address,
```

Comment:
Ideally mock data should be moved to some special files with 'Mock' in the name. This looks very confusing

Reply:
Moved to mock fabric